system/settings/rbp.xml

```diff
@@ -72,6 +72,16 @@
       </group>
     </category>
     <category id="audiooutput">
+      <group id="0">
+        <setting id="audiooutput.useanalogamplifier" type="boolean" label="38122" help="38123">
+          <level>0</level>
+          <default>false</default>
+          <control type="toggle" />
+          <updates>
```

Comment:
The `<update>` part/section might have been needed for the setting you used as reference but it's not necessary for your new setting, as there is no old value to migrate/update. So just remove this section.